public/app/features/transformers/editors/SplitByTransformerEditor.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback } from 'react';
+
+import {
+  DataTransformerID,
+  FieldNamePickerConfigSettings,
+  PluginState,
+  StandardEditorsRegistryItem,
+  standardTransformers,
+  TransformerRegistryItem,
+  TransformerUIProps,
+} from '@grafana/data';
+import { SplitByTransformerOptions } from '@grafana/data/src/transformations/transformers/splitBy';
+import { FieldValidationMessage, InlineField } from '@grafana/ui';
+
+import { FieldNamePicker } from '../../../../../packages/grafana-ui/src/components/MatchersUI/FieldNamePicker';
```

Comment:
Feel free to drop because I'm probably missing something obvious (I watched the Pluralsight typescript course in February and then forgot most of it because I became a dataframer), but I noticed that this import looks a bit degenerate compared to the others. Is there an easy fix for that?

Reply:
Yeah I thought this was odd, too, but I was following the precedent from this file:
https://github.com/ni/grafana/blob/1ae3d007e4578408ecf8f20ab3040cd32363e7f7/public/app/features/transformers/editors/ConvertFieldTypeTransformerEditor.tsx#L17
There may be an easy fix, but I'd rather stick with whatever upstream Grafana is doing.

Reply:
Makes sense, thanks